modules/installation-guide/examples/system-variables.adoc

```diff
@@ -95,57 +95,55 @@ pass:[<!-- vale off -->]
- `+CHE_INFRA_KUBERNETES_CLIENT_HTTP_CONNECTION__POOL_KEEP__ALIVE__MIN+`,"`+5+`","Keep-alive timeout of the connection pool of the Kubernetes-client shared http client in minutes" 
- `+CHE_INFRA_KUBERNETES_TLS__ENABLED+`,"`+false+`","Creates Ingresses with Transport Layer Security (TLS) enabled In OpenShift infrastructure, Routes will be TLS-enabled" 
- `+CHE_INFRA_KUBERNETES_TLS__SECRET+`,"","Name of a secret that should be used when creating workspace ingresses with TLS Ignored by OpenShift infrastructure" 
- `+CHE_INFRA_KUBERNETES_TLS__KEY+`,"`+NULL+`","Data for TLS Secret that should be used for workspaces Ingresses cert and key should be encoded with Base64 algorithm These properties are ignored by OpenShift infrastructure" 
- `+CHE_INFRA_KUBERNETES_TLS__CERT+`,"`+NULL+`","Datafor TLS Secret that should be used for workspaces Ingresses cert and key should be encoded with Base64 algorithm These properties are ignored by OpenShift infrastructure" 
- `+CHE_INFRA_KUBERNETES_RUNTIMES__CONSISTENCY__CHECK__PERIOD__MIN+`,"`+-1+`","Defines the period with which runtimes consistency checks will be performed. If runtime has inconsistent state then runtime will be stopped automatically. Value must be more than 0 or `-1`, where `-1` means that checks won't be performed at all. It is disabled by default because there is possible {prod-short} Server configuration when {prod-short} Server doesn't have an ability to interact with Kubernetes API when operation is not invoked by user. It DOES work on the following configurations: - workspaces objects are created in the same namespace where {prod-short} Server is located; - cluster-admin service account token is mount to {prod-short} Server pod; It DOES NOT work on the following configurations: - {prod-short} Server communicates with Kubernetes API using token from OAuth provider;" 
- `+CHE_INFRA_KUBERNETES_TRUSTED__CA_SRC__CONFIGMAP+`,"`+NULL+`","Name of cofig map in {prod-short} server namespace with additional CA TLS certificates to be propagated into all user's workspaces. If the property is set on OpenShift 4 infrastructure, and che.infra.openshift.trusted_ca.dest_configmap_labels includes config.openshift.io/inject-trusted-cabundle=true label, then cluster CA bundle will be propagated too." 
- `+CHE_INFRA_KUBERNETES_TRUSTED__CA_DEST__CONFIGMAP+`,"`+ca-certs+`","" 
- `+CHE_INFRA_KUBERNETES_TRUSTED__CA_MOUNT__PATH+`,"`+/public-certs+`","Configures path on workspace containers where the CA bundle should be mount. Content of config map specified by che.infra.kubernetes.trusted_ca.dest_configmap is mounted." 
- `+CHE_INFRA_KUBERNETES_TRUSTED__CA_DEST__CONFIGMAP__LABELS+`,"","Comma separated list of labels to add to the CA certificates config map in user workspace. See che.infra.kubernetes.trusted_ca.dest_configmap property." 
+ `+CHE_INFRA_KUBERNETES_POD_SECURITY__CONTEXT_RUN__AS__USER+`,"`+NULL+`","Defines security context for Pods that will be created by Kubernetes Infra This is ignored by OpenShift infra" 
```

Comment:
missing a period in "Kubernetes Infra This is ignored by OpenShift infra"

Also I see you have "Kubernetes Infra" (uppercase I) and "OpenShift infra" (lowercase i). Might be worth a global f&r on those to make them all consistently cased.